### DISCUSSION TITLE
Contributing and Support

### DISCUSSION THREAD

**Discussion by cepedamarc** [LukeCroteau/RaidExtractor#29] [General]
I'm quite surprised that I am so late to the party on this, but it's precisely the thing I've wanted to do for my auto clicker since the beginning but chose to go a "safer" route by avoiding it altogether. Now that some precedents have been set and it seems to be OK in Plarium's eyes so long as it's nothing malicious or results in unfair advantage to a player (i.e. cheating), I've begun the work that you seem to have already done a lot of the heavy lifting for, of which is more than impressive. By any chance could we correspond via Discord? If so, I am click#5074.

At the very least, I'd like to contribute to this project, if I may, but I'd also like to pick your brain about a few things if you'll allow it. I look forward to hearing back from you either way.

Best regards,
OMGclick.

**Comment by Gcvanasel** [comment]
Hi Luke, 

Awesome project and I would like to support where I can. I have experience in quite a vast spectrum of programming languages so let me know were you need some assistance 

Piece

Chris

**Comment by LukeCroteau** [comment]
@omgmarc @Gcvanasel 
My apologies! I forgot there was even a discussion area for Github projects. It's been so long since I used Github for Day to Day interactions.

Please take a look at the Issues available and see if any trip your interest, or if there are features either of you are looking to implement.

I'm in the OMG discord as a Patreon supporter. 
My discord tag is voltaic#8915 if you'd like to reach out directly as well.
